### PR TITLE
Changing from Admiralawkbar to github

### PR DESCRIPTION
**Pull request by admiralAwkbar**
This PR will cover:
- Moving the underlying image from `admiralawkbar` to `GitHub` on **Docker**

### CODE REVIEW

**Comment by admiralAwkbar** [comment]
@zkoppert pretty easy updates to get this from `admiralawkbar` to `github`
Need to look for early adopters and make sure they are not hard pointed at my old org ;)

File: .automation/cleanup-docker.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 ################################################################################
-############# Cleanup Container on DockerHub @admiralawkbar ####################
+############# Cleanup Image on DockerHub @admiralawkbar ########################
```
Comment (zkoppert) [inline .automation/cleanup-docker.sh:4]: Should the @admiralawkbar reference change to github?

Comment (admiralAwkbar) [reply .automation/cleanup-docker.sh:4]: it was just my name at the top so people could see who wrote the terrible code :)

**Comment by admiralAwkbar** [comment]
@zkoppert no need to update creds, I was just granted access through my SSO

**Pull request merged**